lua/tree-sitter-go-template/init.lua

```diff
@@ -0,0 +1,87 @@
+local M = {}
+
+local lang_name = "gotmpl"
+
+local known_additional_langs = {
+  "helm",
+}
+
+local path = debug.getinfo(1).source:sub(2):match "(.*[/\\])" or "./"
+local parser_config = require("nvim-treesitter.parsers").get_parser_configs()
+
+--- @param additional_langs string[]
+local function configure_filetype(additional_langs)
```

Comment:
This functions seems to be generic, but then it contains details for the helm filetype. Maybe we should just make one `configure_helm_filetype` function instead, which is called, when additional_langs contains "helm". Or `known_additional_langs` could also contain the patterns of the known languages.

Reply:
Maybe this is just a misleading name. I named it after `vim.filetype`. Basically this means "configure filetype detection". Should I just rename the function to `configure_filetype_detection`?